samples/snippets/src/main/java/pubsub/SubscribeSyncExample.java

```diff
@@ -60,12 +60,20 @@ public static void subscribeSyncExample(
 
       // Use pullCallable().futureCall to asynchronously perform this operation.
       PullResponse pullResponse = subscriber.pullCallable().call(pullRequest);
+
+      // Exit the program if the pull response is empty.
```

Comment:
This comment tells me _what_ you are doing (but so does the code). Can we update it to tell me _why_ we are doing?

Reply:
That's a good point. Updated.